src/main/resources/assets/securitycraft/lang/ko_kr.json

```diff
@@ -618,41 +728,70 @@
 	"messages.securitycraft:mrat.cantBind": "당신 소유가 아닌 지뢰는 바인딩할 수 없습니다.",
 	"messages.securitycraft:mrat.bound": "%s에 있는 지뢰는 원격 접속기로 나눕니다.",
 	"messages.securitycraft:mrat.unbound": "원격 접속기에서 %s의 지뢰를 분리합니다.",
-	"messages.securitycraft:srat.noSlots": "이 보초를 바인딩할 빈 슬롯이 더 이상 없습니다!",
-	"messages.securitycraft:srat.cantBind": "자기 소유가 아닌 보초는 바인딩할 수 없습니다.",
-	"messages.securitycraft:srat.bound": "%s의 보초를 원격 접속기에 경계합니다.",
-	"messages.securitycraft:srat.unbound": "원격 접속기에서 %s에서 보초의 바인딩을 해제합니다.",
+	"messages.securitycraft:srat.noSlots": "이 센트리 건를 바인딩할 빈 슬롯이 더 이상 없습니다!",
+	"messages.securitycraft:srat.cantBind": "자기 소유가 아닌 센트리 건는 바인딩할 수 없습니다.",
+	"messages.securitycraft:srat.bound": "%s의 센트리 건를 원격 접속기에 경계합니다.",
+	"messages.securitycraft:srat.unbound": "원격 접속기에서 %s에서 센트리 건의 바인딩을 해제합니다.",
 	"messages.securitycraft:universalOwnerChanger.noName": "이 아이템에 새 블록 소유자의 이름을 지정해야 합니다. 모루를 사용하여 이름을 바꿉니다.",
 	"messages.securitycraft:universalOwnerChanger.cantChange": "이 블록은 소유자를 포함할 수 없습니다. 적절한 세큐리티크래프트 블록을 마우스 오른쪽 버튼으로 클릭하세요.",
 	"messages.securitycraft:universalOwnerChanger.notOwned": "이 블록은 사용자의 블록이 아니므로 소유자를 변경할 수 없습니다.",
-	"messages.securitycraft:universalOwnerChanger.changed": "%s는 이제 새로운 주인이 되었습니다.",
+	"messages.securitycraft:universalOwnerChanger.changed": "%s은(는) 이제 새로운 주인이 되었습니다.",
 	"messages.securitycraft:universalOwnerChanger.noBlockClaiming": "소유자가 없으면 블록을 청구할 수 없습니다.",
+	"messages.securitycraft:universalOwnerChanger.briefcase.notOwned": "이 서류 가방은 귀하의 서류 가방이 아니므로 소유자를 변경할 수 없습니다.",
 	"messages.securitycraft:irc.connected": "세큐리티크래프트의 v1.8.3에서는 IRC 채팅이 더 이상 지원되지 않습니다. 도움이 필요한 경우 세큐리티크래프트의 디스코드 #help 채널을 참조해 주세요!",
-	"messages.securitycraft:cageTrap.captured": "%1$s는 %2$s에서 잡혔습니다.",
+	"messages.securitycraft:cageTrap.captured": "%1$s은(는) %2$s에서 잡혔습니다.",
 	"messages.securitycraft:frame.rightclick": "바인딩된 모니터로 마우스 오른쪽 버튼을 클릭하여 봅니다.",

```

Comment:
This was language line was completely changed in en_us, please translate to the new message.

Reply:
I modified it.

Reply:
"messages.securitycraft:frame.rightclick" is still the same as before. It should be "Viewing a camera monitor in a frame is currently not possible because LookingGlass has not been ported to any version newer than 1.7.10. In a future update, this feature might return without depending on any library mod like LookingGlass, however it is unknown when exactly."